gui/lib/daemon/error_screen.dart

```diff
@@ -247,3 +156,22 @@ ErrorData _dataForAccountError(WidgetRef ref) {
     ),
   );
 }
+
+CopyField _buildCopyFieldForSocketNotFound() {
+  if (Platform.environment.containsKey('SNAP_NAME')) {
```

Comment:
Could this check be reused? I see the same thing is used in `snap_permissions_provider.dart`